aldryn_translation_tools/static/css/admin/all-translations-mixin.css

```diff
@@ -0,0 +1,23 @@
+a.lang-code {
```

Comment:
@FinalAngel will be very angry when he sees all that css code indented with only two spaces :)